app/templates/frameworks/_framework_actions.html

```diff
@@ -27,17 +27,11 @@
 <li class="browse-list-item">
   <a class="browse-list-item-link"
      href="{{ url_for('.supplier_details') }}">
-    <span>Enter your company details</span>
+    <h2>1. Enter your company details</h2>
   </a>
-  {% if not company_details_complete and counts.complete %}
-  <div class="browse-list-item-status-angry">
-    <p class="browse-list-item-status-title">
-    <strong>No services will be submitted because you haven’t completed your company details</strong>
-    </p>
-  </div>
-  {% elif not company_details_complete %}
+  {% if not application_company_details_confirmed %}
   <div class="browse-list-item-status-quiet">
-    <p class="browse-list-item-status-title">You must add your organisation’s details</p>
+    <p class="browse-list-item-status-title">You must {% if not supplier_company_details_complete %}add{% else %}confirm{% endif %} your organisation’s details</p>
```

Comment:
This might be clearer as `{% if supplier_company_details_complete %}confirm{% else %}add{% endif %}`

Reply:
Yep that's fair enough 👍